doc-content/kogito-docs/src/main/asciidoc/drl/proc_drl-rules-create.adoc

```diff
@@ -196,16 +199,16 @@ import org.kie.kogito.rules.DataSource;
 import org.kie.kogito.rules.DataStream;
 
 declare MortgageRules extends RuleUnitData
-  bankruptcy: DataStream<Bankruptcy> = DataSource.createStream()
-  applicant: DataStream<Applicant> = DataSource.createStream()
-  application: DataStream<LoanApplication> = DataSource.createStream()
+  bankruptcy: DataStore<Bankruptcy> = DataSource.createStore()
+  applicant: DataStore<Applicant> = DataSource.createStore()
+  application: DataStore<LoanApplication> = DataSource.createStore()
```

Comment:
A DataStream is an append only DataSource. Since in at least one case you're doing a remove I changed all in DataStores.

Reply:
Done